packages/fast-components-react-msft/build/create-sketch-library.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Generate the sketch library file for MSFT components.
+ *
+ * To use this script:
+ * 1. Ensure you have the development site running - from the root of this project, run `npm start` and wait until the website is built
+ * 2. run `node build/create-sketch-library.js`
+ * 3. Load the output `fast-dna-msft-design-kit.json` file into sketch using the plugin from `https://github.com/brainly/html-sketchapp/releases/`
+ * 4. Save out the file as a .sketch file to app/fast-dna-msft-design-kit.sketch
+ */
+const extractSymbolLibrary = require("@microsoft/fast-sketch-library").extractSymbolLibrary;
+const fs = require("fs");
+
+const sources = [
+    "http://localhost:7001/components/button/",
+    "http://localhost:7001/components/checkbox/",
+    "http://localhost:7001/components/divider/",
+    "http://localhost:7001/components/heading/",
+    "http://localhost:7001/components/label/",
+    "http://localhost:7001/components/toggle/",
+    "http://localhost:7001/components/typography/",
+    "http://localhost:7001/components/hypertext/",
```

Comment:
Should these instead be made relative?  How would I do this in a production environment?

Reply:
Curious on this as well, as right now I believe the same thing that is rendered into dev is what we compile and render as our production docs site right now.

Reply:
Nope. You point it any URL - if you want to use it in production, you point it at a production URL.